main/main.c

```diff
@@ -35,6 +44,13 @@ void app_main(void)
     Spotify_TaskInit(&SpotifyInterfaceHandler);
     // after this semaphore you can use playback command function in every where !
     if (xSemaphoreTake(IsSpotifyAuthorizedSemaphore, portMAX_DELAY) == pdTRUE)
+    {
         Spotify_SendCommand(GetNowPlaying);
+        vTaskDelay((pdMS_TO_TICKS(SEC * 5)));
+        Spotify_SendCommand(Pause);
```

Comment:
if increase delay between your request , 
![image](https://github.com/Behn4m/simple_server_Spotify/assets/104908459/2ffe3fe7-ba6d-4864-b871-48c528ba0433)
get solve .

Reply:
or add mechanism for preventing two time defining dynamic variable